Makefile

```diff
@@ -8,8 +7,8 @@ TEST_FILES=$(shell find test/*.coffee)
 # for now, exactly the same as .all
 .pretestem: .all
 
-.app: $(APP_FILES)
-	node_modules/.bin/browserify $(APP_FILES) -o dist/twine.js -t coffeeify
+.app:
+	node_modules/.bin/coffee -p lib/assets/javascripts/twine.coffee > dist/twine.js
```

Comment:
That issue got closed with a commit that looks like an option got added: https://github.com/jashkenas/coffeescript/commit/2b03fa9077cea31b25cb23518d8af59fe5b80fed